packages/augur-ui/src/modules/markets-list/components/markets-view.tsx

```diff
@@ -115,6 +118,7 @@ export default class MarketsView extends Component<
       includeInvalidMarkets,
     } = this.props;
     const { filter, sort, limit, offset } = this.state;
+    this.props.updateMarketsListSearching(true);
```

Comment:
`updateMarketsListSearching` isn't a good name. Seems like `clearCategoryMeta` would be a better description

Reply:
👍